HomeWork_One/Quiz_Template/src/components/inputs/input.tsx

```diff
@@ -0,0 +1,35 @@
+import "./Input.css";
+import { useDispatch } from "react-redux";
+import { setOptions } from "../../redux/configuration/configSlice.ts";
+
+function Input({ name }) {
+  const dispatch = useDispatch();
+  const handleChange = (event) => {
+    const name = event.target.name;
+    const value = event.target.value;
+    dispatch(setOptions({ name, value }));
+  };
```

Comment:
this function can be placed in a parent component
then it allows you to avoid duplications
Duplication: HomeWork_One/Quiz_Template/src/components/inputs/Select.tsx